docs/streams/developer-guide/app-reset-tool.html

```diff
@@ -0,0 +1,236 @@
+                upstream application.</li>
+            <li>Reset the local environment of your application instances.  It is your responsibility to delete the local
+                state on any machine on which an application instance was run.  See the instructions in section
+                <a class="reference internal" href="#streams-developer-guide-reset-local-environment"><span class="std std-ref">Step 2: Reset the local environments of your application instances</span></a> on how to do this.</li>
+        </ul>
+        <dl class="docutils">
+            <dt>Prerequisites</dt>
+            <dd><ul class="first last">
+                <li><p class="first">All instances of your application must be stopped. Otherwise, the application may enter an invalid state, crash, or produce incorrect results. You can verify whether the consumer group with ID <code class="docutils literal"><span class="pre">application.id</span></code> is still active by using <code class="docutils literal"><span class="pre">bin/kafka-consumer-groups</span></code>.</p>
+                </li>
+                <li><p class="first">Use this tool with care and double-check its parameters: If you provide wrong parameter values (e.g., typos in <code class="docutils literal"><span class="pre">application.id</span></code>) or specify parameters inconsistently (e.g., specify the wrong input topics for the application), this tool might invalidate the application&#8217;s state or even impact other applications, consumer groups, or your Kafka topics.</p>
+                </li>
+                <li><p class="first">You should manually delete and re-create any intermediate topics before running the application reset tool. This will free up disk space in Kafka brokers.</p>
+                </li>
+                <li><p class="first">You should delete and recreate intermediate topics before running the application reset tool, unless the following applies:</p>
+                    <blockquote>
+                        <div><ul class="simple">
+                            <li>You have external downstream consumers for the application&#8217;s intermediate topics.</li>
+                            <li>You are in a development environment where manually deleting and re-creating intermediate topics is unnecessary.</li>
+                        </ul>
+                        </div></blockquote>
+                </li>
+            </ul>
+            </dd>
+        </dl>
+        <div class="section" id="step-1-run-the-application-reset-tool">
+            <h2>Step 1: Run the application reset tool<a class="headerlink" href="#step-1-run-the-application-reset-tool" title="Permalink to this headline"></a></h2>
+            <p>Invoke the application reset tool from the command line</p>
+            <div class="highlight-bash"><div class="highlight"><pre><span></span>&lt;path-to-confluent&gt;/bin/kafka-streams-application-reset
```

Comment:
This is a meta comment: we need to replaces these contents such as path names and package names that includes `confluent`. Instead we have to use the AK corresponding ones.

Reply:
i deleted, per this conversation https://confluent.slack.com/archives/C1B9SEJES/p1511386083000124?thread_ts=1511313343.000036&cid=C1B9SEJES